### DISCUSSION TITLE
Item lvl 128 badge stuff in T5 bracket

### DISCUSSION THREAD

**Discussion by Johnbeacon** [chromiecraft/chromiecraft#6264] [Suggestions]
It will be a chance to get item level 128 badge stuff (at least not haste/arp gear) in SSC -TK bracket? Item level it's the same and most of these items will be useless in ZA

**Comment by balleny** [comment]
they were always meant as a catch-up.
dividing the items and taking out haste or ArP gear would be a bad decision too, because some classes would potentially not get something at all while others still get plenty of options.
lastly, badges are not intended to give loot of the current bracket but a bracket below in tbc.
